root/edit/search_macros.tt

```diff
@@ -384,6 +386,12 @@
   </span>
 [% END %]
 
+[% MACRO predicate_edit_subscription(field, field_contents) WRAPPER wfield predicate="edit_subscription" %]
+  [% operators([ [ 'subscribed', l('is in my subscriptions') ]
+                 [ 'not_subscribed', l('is not in my subscriptions') ]
```

Comment:
To follow and complete the previous comment:
```suggestion
  [% operators([ [ 'subscribed', l('is directly affected') ]
                 [ 'not_subscribed', l('is not directly affected') ]
```

Also suggested to add “directly” in case the edit search doesn’t match the same edits as the subscriptions email.

Reply:
> I don't understand directly affected

Some edits are related to entities but not directly affecting these. For example, editing the work composed by an artist is not directly affecting the artist, but it still is related and appears in artist’s editing history.

> nor the relation with the subscription email.

I wondered if the edits matched for building the subscriptions email are the same as the edits matched by this search.

Anyway, my suggestion is no longer relevant since _Edit_ is about to be replaced with _Edited entity_ rather than _Subscribed entity_.

Reply:
Ok now I understand, thanks!